docs/CONTRIBUTING.adoc

```diff
@@ -179,3 +179,38 @@ To submit a pull request:
 . Click the _Create pull request_ button.
 
 The pull request will be built in link:https://ci.jenkins.io[ci.jenkins.io].
+
+== Releases
+
+The releases can be generated by using the `release.sh` script at the root of the repository.
+This scripts uses `git`, `gh`, `jq` and `mvn` to generate the release.
+The script also requires Bash version to be greater than 4.
+Make sure you have all those tools on your system.
+
+You can run the script like this:
+
+[source bash]
+----
+bash release.sh
+----
+
+WARNING: to be able to perform a release, you **must** have write permission on the project repository.
+
+The script will pick the version from the GitHub Release in draft which is prepared by `release-drafter`.
+
+You can pass a version as the first argument of the script, but only if you know exactly what you are doing.
+Then, run the script like:
+
+[source bash]
+----
+bash release.sh 9.9.9
+----
+
+The project doesn't publish Java binary.
+The release is "only" generating a Git tag, changing the versions in the `pom.xml` files.
+The tag is picked up by infra.ci.jenkins.io to build the docker image.
+
+When both the docker image and the GitHub Release are available, Infra team has a Jenkins job using `updatecli` which will create a pull request on link:https://github.com/jenkins-infra/helm-charts[].
+This pull request should be reviewed carefully.
+Once it is merged, a deployment will be trigger through a pull request on link:https://github.com/jenkins-infra/kubernetes-management[] repository.
+Only when this second pull request is merge that the new release is deployed on the infra system and available at link:https://plugin-health.jenkins.io[].
```

Comment:
```suggestion
The new release is *only* deployed on the infra system and available at link:https://plugin-health.jenkins.io[] when this second pull request is merged.
```
Suggestion to re-order the syntax so that it reads a little better/more smoothly.